### ISSUE TITLE
My recent - should show only according to the tab you are in 

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#1822]
If you are in backgrounds, only recent backgrounds should show.
If you in images, then only recent images should be seen.

**Comment by chayaml** [comment]
also by overlays, only recent overlays should show.

**Comment by chayaml** [comment]
definitely! Mistakenly thought recents images appeared by overlays but this makes sense!
